<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Please enter the date of the experiment:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="date">
        <date>
         <year>2020</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Please enter the time of the experiment:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTimeEdit" name="timeEdit">
       <property name="displayFormat">
        <string>hh:mm:ss</string>
       </property>
       <property name="time">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="retLabel">
       <property name="text">
        <string>Please enter your desired retention time:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="retTimeBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="flexLabel">
       <property name="text">
        <string>Please enter your desired wiggle time:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="flexTimeBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="dirLabel">
       <property name="text">
        <string>Please enter the directory of the XLSX file:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="directoryLineEdit">
       <property name="text">
        <string>C:/Users/...</string>
       </property>
       <property name="placeholderText">
        <string extracomment="xlsx directory"/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="signalLabel">
       <property name="text">
        <string>Please choose your desired signal:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="signalComboBox">
       <property name="currentText">
        <string>210 nm</string>
       </property>
       <item>
        <property name="text">
         <string>210 nm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>360 nm</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
